Packs/PrismaAccess/Integrations/PrismaSASE/PrismaSASE.yml

```
@@ -0,0 +1,1891 @@
+      name: predefined_url_list
+      required: false
+      description: Dont know yet
+      predefined:
+      - panw–auth-portal-exclude-list
+    - default: false
+      isArray: false
+      name: source_url
+      description: Dont know yet
+    - auto: PREDEFINED
+      isArray: false
+      name: frequency
+      required: false
+      description: Dont know yet
+      predefined:
+      - five_minute
+      - hourly
+      - daily
+      - weekly
+      - monthly
+    - default: false
+      isArray: false
+      name: frequency_hour
+      description: The frequency hour
+    - auto: PREDEFINED
+      isArray: false
+      name: day_of_week
+      required: false
+      description: The day of the week
+      predefined:
+      - monday
+      - tuesday
+      - wednesday
+      - thursday
+      - friday
+      - saturday
+      - sunday
+    - default: false
+      isArray: false
+      name: day_of_month
+      description: The day of the month. Possible values are between 1 and 31.
+    - default: false
+      isArray: false
+      name: exception_list
+      description: The user can exclude certain addresses from the list depending on the type of list.
+    - description: Tenant services group ID. If not provided, the tsg_id integration parameter will be used as the default.
+      name: tsg_id
+    deprecated: false
+    execution: false
+    description: Update an existing dynamic list.
+    name: prisma-sase-external-dynamic-list-update
+    outputs:
+    - contextPath: PrismaSase.ExternalDynamicList.id
+      description: The external dynamic list id.
+      type: String
+    - contextPath: PrismaSase.ExternalDynamicList.name
+      description: The external dynamic list name.
+      type: String
+    - contextPath: PrismaSase.ExternalDynamicList.folder
+      description: The external dynamic list folder.
+      type: String
+    - contextPath: PrismaSase.ExternalDynamicList.description
+      description: The external dynamic list description.
+      type: String
+    - contextPath: PrismaSase.ExternalDynamicList.type
+      description: The external dynamic list type.
+      type: String
+    - contextPath: PrismaSase.ExternalDynamicList.source
+      description: The external dynamic list source.
+      type: String
+    - contextPath: PrismaSase.ExternalDynamicList.frequency
+      description: The external dynamic list frequency.
+      type: String
+  - arguments:
+    - default: false
+      isArray: false
+      name: id
+      required: true
+      description: The external dynamic list id.
+    - description: Tenant services group ID. If not provided, the tsg_id integration parameter will be used as the default.
+      name: tsg_id
+    deprecated: false
+    execution: false
+    description: Delete a specific dynamic list.
+    name: prisma-sase-external-dynamic-list-delete
+  - arguments:
+    - auto: PREDEFINED
+      default: false
+      defaultValue: Shared
+      description: Prisma sase folder location.
```

```suggestion
      description: Prisma SASE folder location.
```
Is this the folder location or folder type/name?